tests/emukit/bayesian_optimization/test_mean_plugin_expected_improvement.py

```diff
@@ -0,0 +1,69 @@
+import GPy
```

Comment:
since the test is for the acquisition function, it would be ideal if you could remove GPy here at all, and just do with the mocks